webserver/templates/datasets/macros.html

```diff
@@ -3,28 +3,33 @@
   <div id="dataset-header">
 
     <div class="row">
-      <div {% if is_author %}class="col-md-7 col-lg-8"{% else %}class=col-md-12'{% endif %}>
+      <div {% if is_author %}class="col-md-6 col-lg-7"{% else %}class='col-md-6'{% endif %}>
         <h2 class="page-title">Dataset "{{ dataset['name'] }}"</h2>
       </div>
-      {% if is_author %}
-        <div id="control-buttons" class="col-md-5 col-lg-4">
-          <a class="btn btn-success" title="Evaluate this dataset"
-             href="{{ url_for('datasets.evaluate', dataset_id=dataset['id']) }}">
-            <span class="glyphicon glyphicon-tasks" aria-hidden="true"></span>
-            Evaluate
-          </a>
-          <a class="btn btn-primary" title="Edit this dataset"
-             href="{{ url_for('datasets.edit', dataset_id=dataset['id']) }}">
-            <span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>
-            Edit
-          </a>
-          <a class="btn btn-danger" title="Delete this dataset"
-             href="{{ url_for('datasets.delete', dataset_id=dataset['id']) }}">
-            <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
-            Delete
-          </a>
-        </div>
-      {% endif %}
+          {% if is_author %}
+              <a class="btn btn-success pull-right" title="Evaluate this dataset"
+                href="{{ url_for('datasets.evaluate', dataset_id=dataset['id']) }}">
+                <span class="glyphicon glyphicon-tasks" aria-hidden="true"></span>
+                Evaluate
+              </a>
+              <a class="btn btn-primary pull-right" title="Edit this dataset"
+                href="{{ url_for('datasets.edit', dataset_id=dataset['id']) }}">
+                <span class="glyphicon glyphicon-pencil" aria-hidden="true"></span>
+                Edit
+              </a>
+              <a class="btn btn-danger pull-right" title="Delete this dataset"
+                href="{{ url_for('datasets.delete', dataset_id=dataset['id']) }}">
+                <span class="glyphicon glyphicon-trash" aria-hidden="true"></span>
+                Delete
+              </a>
+            {% endif %}
+
+        <a class="btn btn-info pull-right" title="Download this dataset" 
+          href="{{ url_for('datasets.download_csv', dataset_id=dataset['id']) }}">
+          <span class="glyphicon glyphicon-download-alt" aria-hidden="true"></span>
+          Download
```

Comment:
Let's make this a bit more specific so that we don't confuse it with #267. Let's call it "Download schema"